doc/rst/developer/bestPractices/Deprecation.rst

```diff
@@ -1,9 +1,46 @@
 Chapel's Deprecation Policy
 ===========================
 
-Chapel's general policy for deprecation is to:
+When removing a feature in Chapel, the feature is to be
+deprecated and supported for at least 1 release cycle.
 
-- Provide a compiler warning (or error) when a deprecated feature is used
-- Continue to document and support the deprecated feature for 1 release cycle
-  - The documentation should contain a pointer to the preferred feature
-  - This holds true for modules as well
+Major feature removals can be supported for more than 1 release cycle.
+
+Non-public interface features can be removed or modified without deprecation.
+
+This is the deprecation process:
+
+1. Provide a compiler warning (or error, if necessary) when a deprecated feature is used
```

Comment:
Remove the word "compiler"?  Still think we need to avoid implying that's the only place the message can come from.